var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -14,6 +14,8 @@ class Hypre(Package):
 
     # hypre does not know how to build shared libraries on Darwin
     variant('shared', default=sys.platform!='darwin', description="Build shared library version (disables static library)")
+    # SuperluDist have conflicting headers with those in Hypre
+    variant('internal-superlu', default=True, description="Use internal Superlu routines")
```

Comment:
Sorry, I *did* mean `hypre` above.  This makes sense -- `internal-superlu` is probably fine as it is currently.  I should  ask Ulrike and Rob about this -- currently they're all trying to collaborate on a release of the [`xSDK`](https://ideas-productivity.org/focus-a…), which includes `hypre`, `petsc`, `superlu`, `superlu_dist`, etc.  I'm curious if this option is going to stay the same.